src/main/java/stream/flarebot/flarebot/commands/general/ShardInfoCommand.java

```diff
@@ -41,14 +42,9 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
             row.add(String.valueOf(jda.getVoiceChannels().stream().filter(vc -> vc.getMembers().contains(vc.getGuild()
                     .getSelfMember())).count()));
             table.add(row);
-            // TODO: Replace this hotfix with pagination
-            if (table.size() == 20) {
-                channel.sendMessage(MessageUtils.makeAsciiTable(headers, table, null, "swift")).queue();
-                table = new ArrayList<>();
-            }
         }
         if (table.size() > 0) {
-            channel.sendMessage(MessageUtils.makeAsciiTable(headers, table, null, "swift")).queue();
+            PaginationUtil.sendPagedMessage(channel, PaginationUtil.buildPagedTable(headers, table, 20), 0);
```

Comment:
Can we please make a better system than making lists of lists and crap for rows. It's so bad. Maybe a PagedTableBuilder? setColums(content).addRow(content).addRow(content) etc.

Reply:
Sure